test/test_permeation_problem.py

```diff
@@ -96,23 +89,43 @@ def siverts_law(T, S_0, E_S, pressure):
 
         my_model.solver.solve(u)
 
+        mobile_xdmf.write_function(u, t)
+
         # post process
-        surface_flux = form(my_model.D * dot(grad(u), n) * ds(2))
+        surface_flux = form(my_model.D * dot(grad(u), n) * my_model.ds(2))
         flux = assemble_scalar(surface_flux)
         flux_values.append(flux)
         times.append(t)
 
-        # export
-        np.savetxt("outgassing_flux.txt", np.array(flux_values))
-        np.savetxt("times.txt", np.array(times))
-
-        mobile_xdmf.write_function(u, t)
-
         # update previous solution
         mobile_H.prev_solution.x.array[:] = u.x.array[:]
 
     mobile_xdmf.close()
 
+    # analytical solution
+    S = 4.02e21 * exp(-1.04 / F.k_B / float(temperature))
```

Comment:
The pre-exp and activation energy are duplicated (defined here and on line 67
```python
surface_conc = siverts_law(T=temperature, S_0=4.02e21, E_S=1.04, pressure=100)
```

Same goes for the temperature and mesh length. 

Can you store all these in variables to avoid code duplication?